### ISSUE TITLE
"Error during post message handler" - If I open a document from "recommendations" header

### ISSUE THREAD

**Issue by HolgerHees** [nextcloud/richdocuments#2839]
**Describe the bug**
If I open a document from "recommendations" header (last opened documents), I get the following javascript errors.

```
Error during post message handler 
Object { msgId: "App_LoadingStatus", args: {…}, deprecated: false }
 TypeError: model is null
    _updateDetailsView merged-index.js:4826
    getFileModel FilesAppIntegration.js:196
    initAfterReady FilesAppIntegration.js:79
    postMessageHandler Office.vue:236
    handlePostMessage postMessage.tsx:91
    handlePostMessage postMessage.tsx:85
    e postMessage.tsx:29
    e postMessage.tsx:28
    <anonymous> Office.vue:85
    <anonymous> viewer.js:65
    <anonymous> viewer.js:65
```

The document itself is editable.

If I go to the folder where the document is stored and open it from there, everything works fine without any error.

I use richdocuments version 7.1.1 and nextcloud version 25.0.4

**Comment by siccovansas** [comment]
I see the same error using Nextcloud 26.0.1 when I open a Collabora document using the 'Recommendations' bar/header on the homepage of the Files app. The document opens though and is editable. When I close the document I end up seeing nothing, only the wallpaper. When I open de document from its directory the error is not shown and closing the document shows the directory as expected.

**Comment by joshtrichards** [comment]
Duplicate of #2933. Will be fixed via #3010 in next release.
